west.yml

```
@@ -11,9 +11,7 @@
 # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/west/index.html
 
 manifest:
-  # This west.yml requires west 0.9 or later, because the "submodules"
-  # feature used below was introduced then.
-  version: 0.9
+  version: 0.10
```

This won't work; you need the quotes.

```suggestion
  version: "0.10"
```

Otherwise it is the floating point number 0.1, which does not cast to a valid version as a string.

---

Thanks! Fixed.